src/test/scala/chiseltest/iotesters/BlackBoxVerilogDeliverySpec.scala

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.iotesters
+
+import chisel3._
+import chisel3.util.{HasBlackBoxInline, HasBlackBoxResource}
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+class BBAddOne extends HasBlackBoxInline {
+  val io = IO(new Bundle {
+    val in = Input(UInt(16.W))
+    val out = Output(UInt(16.W))
+  }).suggestName("io")
+  setInline("BBAddOne.v",
+  """
+    |module BBAddOne(
+    |    input  [15:0] in,
+    |    output reg [15:0] out
+    |);
+    |  always @* begin
+    |  out <= in + 1;
```

Comment:
```suggestion
    |    out = in + 1;
```
This makes a pretty ugly warning message go away. Test seems to pass anyway. But I don't know quite enough verilog to know if this is the right thing to do